tools/cldr-apps/pom.xml

```diff
@@ -178,18 +178,15 @@
 						</includes>
 					</configuration>
 				</plugin>
-				<plugin>
+				<!-- <plugin>
```

Comment:
What's the rationale for commenting this out rather than just deleting it? Unless there's a really good reason -- like a plan to restore the code in the near future -- I think deletion is better. I've been studying the book Building Maintainable Software by Joost Visser, which includes a rule "Leave No Code in Comments Behind"

Reply:
thanks for the recommend! I think with source control, the old version's just a click away. We may want some more parameters here someday.